src/structures/game_round.ts

```diff
@@ -346,9 +349,16 @@ export default class GameRound extends Round {
             typosAllowed,
         );
 
+        if (this.songStartedAt === null) {
+            logger.error(
+                `uid: ${playerID}, guess: ${guess}, createdAt: ${createdAt} | songStartedAt is unexpectedly null`,
+            );
+            this.songStartedAt = this.startedAt;
```

Comment:
Why is this a valid guess if the song hasn't started?

Reply:
I needed a default value to put into these variables since it can be null. With the current logic, we shouldn't be able to enter this branch since when `songStartedAt` is null, `storeGuess` never gets called. Should I replace this (and the other commented section) as `songStartedAt!` on the variable instead?

Reply:
You don't need a default value, they are separate variables for a reason. Just need to handle the null properly, in this case, if it's null then its clearly not a valid guess that we should store